lib/models/survey_info.dart

```diff
@@ -1,6 +1,8 @@
 import 'package:object_mapper/object_mapper.dart';
+import 'package:equatable/equatable.dart';
 
-class SurveyInfo with Mappable {
+// ignore: must_be_immutable
+class SurveyInfo extends Equatable with Mappable {
   String? title;
   String? description;
   String? coverImageUrl;
```

Comment:
coverImageUrl can be appended with `l` (large) at the end of the URL to provide you a better quality image.

I.e: `https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_` ->  `https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_l`

Reply:
@sleepylee updated 👍

Reply:
CI failed 😂

Reply:
@sleepylee fixed 😓